MyLibrary/MyLibraryApp.API/Controllers/CategoriesController.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using MyLibraryApp.Application.Services.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyLibraryApp.API.Controllers
+{
+    [ApiController]
+    [Route("categories")]
+
+    public class CategoriesController :ControllerBase
+    {
+        private readonly ICategoryService categoryService;
+        public CategoriesController(ICategoryService categoryService)
+        {
+            this.categoryService = categoryService;
+        }
+
```

Comment:
Check this file for missing/too many new lines, please. 😄 It makes the code harder to read if the spacing part is not correct.